### ISSUE TITLE
upgrading to 14.0.2 - .tsx: Unexpected token, expected "}"

### ISSUE THREAD

**Issue by furier** [dotansimha/graphql-code-generator#923]
Hi, after upgrading to **14.0.2** I get this  error when i run the command `gql-gen`, but the typescript is perfectly valid. :S

```
> gql-gen

  √ Parse configuration
  > Generate outputs
    > Generate src/app/shared/models/graphql/types.ts
      √ Load GraphQL schemas
      × Load GraphQL documents
        →   35 |       ],
        Generate
    √ Generate src/app/shared/models/graphql/services.ts


     We found 1 errors

     Failed to generate src/app/shared/models/graphql/types.ts
SyntaxError: C:\Users\<username>\source\repos\<projectname>\MyProject.Web\.tsx: Unexpected token, expected "}" (32:14)

30 |   static forRoot(): ModuleWithProviders {
31 |     return <ModuleWithProviders>{
> 32 |       ngModule: CoreModule,
    |               ^
33 |       providers: [
34 |         ...NB_CORE_PROVIDERS,
35 |       ],
```

**codegen.yml**
```
schema:
  - 'schema.json'
overwrite: true
config:
  avoidOptionals: false
generates:
  src/app/shared/models/graphql/types.ts:
    documents:
      - './src/app/**/*.ts'
    plugins:
      - 'typescript-common'
      - 'typescript-client'
      - 'typescript-server'
  src/app/shared/models/graphql/services.ts:
    documents:
      - './src/app/**/*.gql'
    plugins:
      - 'typescript-common'
      - 'typescript-client'
      - 'typescript-apollo-angular'

```

**Comment by wesselvdv** [comment]
I am also encountering this, I tackled it by moving all my queries to files that have the same name. (eq. queries.ts in every directory that has files that use graphql.)

**Comment by furier** [comment]
Thats not something I am willing to do... So I downgraded to the 4.0.1.alpha version

**Comment by kamilkisiela** [comment]
@furier @wesselvdv could you check `0.15.0-alpha.7df98bdf`?

**Comment by furier** [comment]
@kamilkisiela not getting the error with: `0.15.0-alpha.7c1a7d2a`

**Comment by kamilkisiela** [comment]
We changed the way we extract documents from ts and js files. Before that we used RegExp, now we use [the library](https://github.com/DAB0mB/graphql-tag-pluck) that our teammate wrote. It uses AST to find stuff based on the default export of `graphql-tag`.

I'll release a v0.14.4 soon.

**Comment by kamilkisiela** [comment]
`v0.14.4` released

**Comment by furier** [comment]
@kamilkisiela I have tried both 0.14.4 and 0.14.5 and they both continue to give me the exact same issue, deleted node_modules and installed packages again, just to make sure and no change :(

**Comment by kamilkisiela** [comment]
Could you also remove `yarn.lock`?

**Comment by furier** [comment]
on it...

**Comment by furier** [comment]
still same issue @kamilkisiela

**Comment by furier** [comment]
rolling back to `0.15.0-alpha.7c1a7d2a` in the meantime

**Comment by kamilkisiela** [comment]
Reproduced and fixed in #982

**Comment by kamilkisiela** [comment]
Released in `v0.15.0`
